db/migrate/20191216200333_add_file_size_to_bulk_download.rb

```diff
@@ -0,0 +1,5 @@
+class AddFileSizeToBulkDownload < ActiveRecord::Migration[5.1]
+  def change
+    add_column :bulk_downloads, :output_file_size, :integer
```

Comment:
please add a comment describing what this should contain and when `nil` is a valid value

Reply:
note these comments will be passed on to `schema.rb` for future reference, and to SQL definition

Reply:
Done!